src/app/public/matchers/matchers.ts

```diff
@@ -163,6 +170,38 @@ const matchers: jasmine.CustomMatcherFactories = {
         return result;
       }
     };
+  },
+
+  toHaveResourceText(): jasmine.CustomMatcher {
+    return {
+      compare(
+        actual: string,
+        name: string,
+        args: { name: string, args: any[] },
+        callback: () => void = () => {}
+      ): jasmine.CustomMatcherResult {
+
+        let skyAppResourcesService: SkyAppResourcesService = TestBed.get(SkyAppResourcesService);
```

Comment:
Curious, but what happens if the consumer isn't using `SkyAppResourcesService` and they run this matcher?

Reply:
```
Error: StaticInjectorError[SkyAppResourcesService]: 
	  NullInjectorError: No provider for SkyAppResourcesService!
```

which is exactly what you'd get if you tried to use it directly in a test without listing it as a provider.